src/components/AccordionItem/AccordionItem.js

```diff
@@ -121,13 +121,15 @@ export default class AccordionItem extends Component {
         {...other}>
         <Expando
           type="button"
+          aria-expanded={this.state.open}
           className={`${prefix}--accordion__heading`}
-          role="tab"
           onClick={this.handleHeadingClick}>
           <Icon
             className={`${prefix}--accordion__arrow`}
             icon={iconChevronRight}
             description={iconDescription}
+            role={null} // eslint-disable-line jsx-a11y/aria-role
```

Comment:
Do we need the `role` here if we have `aria-hidden`?

Reply:
yup it's because we don't need `role` that it is being set to `null` here, since `Icon` will assign it a default value

Reply:
Makes sense!